Update README (#133)
bow-swift/nef: README.md

.package(url: "https://github.com/bow-swift/nef.git", from: "{version}")
```

> It is an excellent option if you want to use the full nef features in your macOS app, even to build new tooling on top of nef.

Reviewer: ```suggestion
It is an excellent option if you want to use all nef features in your macOS app, even to build new tooling on top of nef.
```

> You can read more about how to use nef library in the [**nef site**](https://nef.bow-swift.io/docs/tutorials/how-to-use-nef-library-/).

&nbsp;

#### 🔌 Using [Xcode Editor Extension](https://github.com/bow-swift/nef-plugin)

> Some of the `nef` features can be used directly in Xcode as an Extension. You can install it directly from [**App Store**](https://apps.apple.com/app/nef/id1479391704?mt=8) or downloading the last binary from the [**releases section**](https://github.com/bow-swift/nef-plugin/releases).

Reviewer: ```suggestion
Some of `nef` features can be used directly in Xcode as an Extension. You can install it directly from [**App Store**](https://apps.apple.com/app/nef/id1479391704?mt=8) or downloading the last binary from the [**releases section**](https://github.com/bow-swift/nef-plugin/releases).
```

<img src="assets/nef-playground.png" height="100">
</p>

And you can use the following option to specify the name for the `nef Playground` that you are creating:

```bash
➜ nef playground --output ~/Desktop --name LatestBowProject
```

It will create an Xcode project with support for the latest available version of Bow, named `LatestBowProject` in your `~/Desktop`. If you open this `nef playground`, you will find an Xcode Playground where you can import Bow or any of its modules, and start trying some of its features.

By default, `nef playground` will be created for iOS platform. If you need to change it, you can use the `--platform` option.

```bash
➜ nef playground --platform osx
```

If you need to take advantage of nef in your Xcode Playgrounds, you can transform your Xcode Playground into a nef Playground using the following command:

```bash
➜ nef playground --playground <Xcode Playground>
```

Where `<Xcode Playground>` is the path to your Xcode Playground.

###

<details>

> <summary>📣 You can create a nef Playground compatible to any different Bow version, branch or commit; even third-parties dependencies</summary>

Reviewer: ```suggestion
<summary>📣 You can create a nef Playground compatible with any different Bow version, branch or commit; even third-party dependencies</summary>
```

&nbsp;

### 🔨 Compiling a nef Playground

> Xcode lets you check for correctness of your Xcode Playground and run it. However, Apple does not give us compiling commands for Xcode Playground as they do for building Xcode projects. It is particularly useful in Continuous Integration when you want to verify that your playgrounds are not broken when the libraries you depend on are updated. `nef` has an option to compile a `nef Playground`. To do this, you can run the following command:

Reviewer: ```suggestion
Xcode lets you check for correctness of your Xcode Playground and run it. However, Apple does not provide us commands to compile an Xcode Playground, as they do for building Xcode projects. It is particularly useful in Continuous Integration when you want to verify that your playgrounds are not broken when the libraries you depend on are updated. `nef` has an option to compile a `nef Playground`. To do this, you can run the following command:
```
